scripts/leetcode_directory_md.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+from urllib.request import Request, urlopen
+from pathlib import Path
+from os import listdir
+import json
+from dataclasses import dataclass

```

Comment:
It is recommended that you sort your imports. In this case, assuming you use the `requests` library, then you would have this sorting:

```py
from dataclasses import dataclass
from os import listdir
from pathlib import Path

import requests
```